ci-operator/config/chetna14manku/serverless-operator/chetna14manku-serverless-operator-skip_option__ocp4.16-rc.yaml

```diff
@@ -0,0 +1,216 @@
+  to: serverless-ingress
+- dockerfile_path: serving/metadata-webhook/Dockerfile
+  inputs:
+    ocp_ubi-minimal_8:
+      as:
+      - registry.ci.openshift.org/ocp/ubi-minimal:8
+    openshift_release_rhel-8-release-golang-1.21-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.21-openshift-4.16
+  to: serverless-metadata-webhook
+releases:
+  latest:
+    release:
+      channel: candidate
+      version: "4.16"
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: serverless-p2p-rosa
+  cron: 0 6 11 2 *
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      CHANNEL_GROUP: candidate
+      FIREWATCH_CONFIG: |
+        {
+          "failure_rules":
+          [
+            {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure AWS", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["!default"]},
+            {"step": "rosa-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["!default"]},
+            {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+            {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
+            {"step": "openshift-*", "failure_type": "all", "classification": "Other", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+            {"step": "mesh-e2e", "failure_type": "test_failure", "classification": "mesh-e2e Test Failure", "jira_project": "SRVCOM", "jira_additional_labels": ["!default","interop-tests"]},
+            {"step": "ui-e2e", "failure_type": "test_failure", "classification": "ui-e2e Test Failure", "jira_project": "SRVCOM", "jira_additional_labels": ["!default","interop-tests"]},
+            {"step": "mesh-e2e", "failure_type": "pod_failure", "classification": "mesh-e2e Test Execution", "jira_additional_labels": ["!default","interop-tests"]},
+            {"step": "ui-e2e", "failure_type": "pod_failure", "classification": "ui-e2e Test Execution", "jira_additional_labels": ["!default","interop-tests"]},
+          ],
+          "success_rules":
+          [
+            {"jira_project": "CSPIT", "jira_additional_labels": ["!default"]}
+          ]
+        }
+      FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.15-lp-p2p","rosa-classic-lp","serverless-p2p"]'
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPINTEROP
+      FIREWATCH_FAIL_WITH_TEST_FAILURES: "true"
+      OCM_LOGIN_ENV: staging
+      OPENSHIFT_VERSION: "4.16"
+      REGION: us-west-2
+      SLEEP_DURATION: 30m
+    test:
+    - ref: wait
+    - as: mesh-e2e
+      cli: latest
+      commands: SKIP_OPERATOR_SUBSCRIPTION=true ROSA=true make mesh-e2e
```

Comment:
I left a comment here: https://github.com/chetna14manku/serverless-operator/commit/750dbec59841177b6db16aaf0f696a0871439f64#r143508832 
We could maybe skip passing the ROSA param

Reply:
Thanks for review, I'll update the serverless-operator code to use the `oc` command to get `rosa` cluster type